control/controldisplay/formatter.go

```diff
@@ -63,37 +59,23 @@ type Formatter interface {
 	FileExtension() string
 }
 
-func GetExportFormatter(exportFormat string) (Formatter, error) {
-	formatter, found := exportFormatters[exportFormat]
-	if !found {
-		return nil, fmt.Errorf("invalid export format '%s' - must be one of %s", exportFormat, exportFormatters.keys())
-	}
-	return formatter, nil
+func GetDefinedExportFormatter(arg string) (Formatter, bool) {
+	formatter, found := exportFormatters[arg]
+	return formatter, found
 }
 
-func GetOutputFormatter(outputFormat string) (Formatter, error) {
-	formatter, found := outputFormatters[outputFormat]
-	if !found {
-		return nil, fmt.Errorf("invalid output format '%s' - must be one of %s", outputFormat, outputFormatters.keys())
+func GetTemplateExportFormatter(arg string, allowFilenameEvaluation bool) (Formatter, string, error) {
```

Comment:
function comment

Reply:
This will go away with the implementation of the CSV and JSON templates